### PR TITLE
New Fix for tasks

### PR DESCRIPTION
**Pull request by UnitOmega13**

### CODE REVIEW

File: .idea/checkstyle-idea.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <entry key="copy-libs" value="true" />
+        <entry key="location-0" value="BUNDLED:(bundled):Sun Checks" />
+        <entry key="location-1" value="BUNDLED:(bundled):Google Checks" />
+        <entry key="scan-before-checkin" value="false" />
+        <entry key="scanscope" value="JavaOnly" />
+        <entry key="suppress-errors" value="false" />
+      </map>
+    </option>
+  </component>
+</project>
```
Comment (Sofasmile) [inline .idea/checkstyle-idea.xml]: add a blank line

File: src/src/ExpressionChecker.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
-class ExpressionChecker {
+public class ExpressionChecker {
 
-	boolean check(double value1, double value2, double value3){
-		return value1 + value2 == value3;
+	public boolean doubleExpression(double a, double b, double c) {
+		return Math.abs((a + b) - c) < 1E-4;
```
Comment (Sofasmile) [inline src/src/ExpressionChecker.java]: move 1E-4 into separate static final variable in the class

File: src/src/Game.java [hunk 3]
```diff
@@ -1,14 +1,8 @@
...
-		Boolean gameBool1 = Boolean.parseBoolean(string1);
-		Boolean gameBool2 = Boolean.parseBoolean(string2);
-		Boolean gameBool3 = Boolean.parseBoolean(string3);
-		Boolean gameBool4 = Boolean.parseBoolean(string4);
-		if ((gameBool1 == gameBool2) & (gameBool3 == gameBool4)) {
-			return true;
-		} else if ((gameBool1 == gameBool3) & (gameBool2 == gameBool4)) {
-			return true;
-		} else return (gameBool1 == gameBool4) & (gameBool2 == gameBool3);
+	public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
```
Comment (Sofasmile) [inline src/src/Game.java:3]: you can make it more simply

**Review by Sofasmile — COMMENTED** [review]
resolve comments

File: src/src/ExpressionChecker.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
-class ExpressionChecker {
+public class ExpressionChecker {
+	static final float checkFloat = 1E-4;
```
Comment (Sofasmile) [inline src/src/ExpressionChecker.java]: add access modifier and read about constant names
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

**Review by Sofasmile — COMMENTED** [review]
good, but make small fixing

File: src/src/ExpressionChecker.java (hunk 4 above)
Comment (UnitOmega13) [reply src/src/ExpressionChecker.java]: Access modifier added and change the name of constant.

**Review by Sofasmile — APPROVED** [review]
good

**Pull request closed without merging**